extensions/wikia/PageShare/PageShareController.class.php

```diff
@@ -14,14 +14,15 @@ public function index() {
 	}
 
 	public function getShareIcons() {
-		global $wgMemc;
+		global $wgMemc, $wgEnablePageShareWorldwide;
 
 		$browserLang = $this->getVal( 'browserLang' );
 		$useLang = $this->getVal( 'useLang' );
+		$title = $this->getVal( 'title' );
 		$shareLang = PageShareHelper::getLangForPageShare( $browserLang, $useLang );
 
 		// If social icons should be enabled for EN users, and language is different than EN return false
-		if ( empty( $wgEnablePageShareWorldwide ) && $shareLang !== PageShareHelper::SHARE_DEFAULT_LANGUAGE ) {
+		if ( empty( $wgEnablePageShareWorldwide ) && ( $shareLang !== PageShareHelper::SHARE_DEFAULT_LANGUAGE ) ) {
```

Comment:
AFAIK, you don't need the inner braces as `!==` precedes `&&` (ref. http://php.net/manual/en/language.operators.precedence.php).